contrib/fuzz/archive_fuzzer.go

```diff
@@ -94,13 +96,16 @@ func FuzzImportIndex(data []byte) int {
 			hdrName := path.Clean(hdr.Name)
 			switch hdrName {
 			case ocispec.ImageLayoutFile:
+				hasLayoutOrManifest = true
 				break
```

Comment:
Since you're in here:

```suggestion
				hasLayoutOrManifest = true
```

Go switch statements (unlike most other C-inspired languages) do not fall-through by default, so the `break` is superfluous.  Can you remove them?

Reply:
The intention here is to break out of the for loop.